packages/wdio-cli/tests/utils.test.ts

```diff
@@ -391,19 +391,54 @@ describe('getCapabilities', () => {
     })
 
     it('should return driver with capabilities for ios', () => {
-        expect(getCapabilities({ option: 'foo.app', deviceName: 'fooName', udid: 'num', platformVersion: 'fooNum' })).toMatchSnapshot()
+        expect(getCapabilities({ option: 'foo.app', deviceName: 'fooName', udid: 'num', platformVersion: 'fooNum' } as any)).toMatchSnapshot()
         expect(getCapabilities({ option: 'ios' } as any)).toMatchSnapshot()
     })
 
     it('should return driver with capabilities for desktop', () => {
         expect(getCapabilities({ option: 'chrome' } as any)).toMatchSnapshot()
     })
+
+    it('should throw config not found error', () => {
+        jest.unmock('@wdio/config')
+        jest.resetModules()
+        const getCapabilities = require('../src/utils').getCapabilities
+        expect(()=>getCapabilities({ option: './test.js', capabilities: 2 } as any)).toThrowErrorMatchingSnapshot()
+    })
+
+    it('should throw capability not provided', () => {
+        jest.unmock('@wdio/config')
+        jest.resetModules()
+        const getCapabilities = require('../src/utils').getCapabilities
+        expect(()=>getCapabilities({ option: './packages/wdio-cli/tests/__testData__/wdio.conf.js' } as any)).toThrowErrorMatchingSnapshot()
+    })
+
+    it('should through capability not found', () => {
+        jest.unmock('@wdio/config')
+        jest.resetModules()
+        const getCapabilities = require('../src/utils').getCapabilities
```

Comment:
This function is already imported, see https://github.com/webdriverio/webdriverio/pull/8156/files#diff-9f7a65a7afdec8311f3413b7434378fcb1535c60ce7e8b6609f297a3e9bb7b20R20